Cargo.lock

```diff
@@ -1332,6 +1332,7 @@ dependencies = [
  "regex-cursor",
  "ropey",
  "rustix",
+ "serde",
```

Comment:
I believe this change and the one to `helix-stdx/Cargo.toml` can be reset since we don't touch that crate in this PR anymore